api.planx.uk/modules/user/controller.ts

```diff
@@ -0,0 +1,34 @@
+import { z } from "zod";
+import { ValidatedRequestHandler } from "../../shared/middleware/validate";
+import { getClient } from "../../client";
+import { ServerError } from "../../errors";
+
+interface UserResponse {
+  message: string;
+}
+
+export const createUserSchema = z.object({
+  body: z.object({
+    firstName: z.string(),
+    lastName: z.string(),
+    email: z.string().email(),
+    isPlatformAdmin: z.boolean().optional().default(false),
+  }),
+});
+
+export type CreateUser = ValidatedRequestHandler<
+  typeof createUserSchema,
+  UserResponse
+>;
+
+export const createUser: CreateUser = async (req, res, next) => {
+  try {
+    const $client = getClient();
+    await $client.user.create(req.body);
+    return res.send({ message: "Successfully created user" });
+  } catch (error) {
+    return next(
+      new ServerError({ message: "Failed to create user", cause: error }),
+    );
+  }
+};
```

Comment:
would we benefit from a "RemoveUser" here too that, rather than delete the actual record, nullifies the email field to lock them out? Probably not crucial yet, but it feels like the tiny missing piece from an otherwise very useful set of user & team management endpoints ! And it would probably encourage better house-keeping around this.

Reply:
Good shout - I'll pick this up in a quick follow on PR.